packages/react-dom/src/events/DOMPluginEventSystem.js

```diff
@@ -995,27 +978,23 @@ export function accumulateEnterLeaveTwoPhaseListeners(
 }
 
 export function accumulateEventHandleNonManagedNodeListeners(
-  dispatchQueue: DispatchQueue,
-  event: ReactSyntheticEvent,
+  reactEventType: DOMEventName,
   currentTarget: EventTarget,
   inCapturePhase: boolean,
-): void {
+): Array<DispatchListener> {
   const listeners: Array<DispatchListener> = [];
 
   const eventListeners = getEventHandlerListeners(currentTarget);
   if (eventListeners !== null) {
-    const targetType = ((event.type: any): DOMEventName);
```

Comment:
Not sure if the discrepancy (the code path above reads `event.nativeEvent.type` instead) was intentional but for now I'm keeping it.